ecommerce_api_client/exceptions.py

```diff
@@ -0,0 +1,7 @@
+# noqa pylint: skip-file
```

Comment:
@jimabramson Any opposition to the imports here? Consumers are slightly cleaner since they don't have to import slumber directly.